src/plugins/dashboard/public/plugin.tsx

```diff
@@ -34,7 +34,6 @@ import type { SpacesPluginStart } from '@kbn/spaces-plugin/public';
 import type { HomePublicPluginSetup } from '@kbn/home-plugin/public';
 import { replaceUrlHashQuery } from '@kbn/kibana-utils-plugin/common';
 import { createKbnUrlTracker } from '@kbn/kibana-utils-plugin/public';
-import type { SavedObjectsStart } from '@kbn/saved-objects-plugin/public';
```

Comment:
It looks like you should also be able to move `savedObjects` into the `requiredBundles` section in `src/plugins/dashboard/kibana.jsonc`

Reply:
thx! let's do both suggestions in a new PR, since this one is green and CI stability hasn't been too great.

Reply:
Followup [here](https://github.com/elastic/kibana/pull/174639)